key.util/src/main/java/org/key_project/util/helper/FindResources.java

```diff
@@ -56,7 +56,7 @@ public static <T> List<Path> getResources(String path, Class<T> clazz)
             Path dir = fs.getPath(path);
             return Files.list(dir).collect(Collectors.toList());
         }
-        throw new UnsupportedOperationException("Cannot list files for URL $dirURL");
```

Comment:
Please revert these unrelated changes.

Reply:
Sure, how did they get in in the first place?!